docs-content/general/6_product-features/6_metrics/2_metrics-tutorials/3_user-engagement.md

```diff
@@ -0,0 +1,56 @@
+---
+toc: User Engagement
+title: Creating User Engagement Metrics
+slug: user-engagement
+---
+
+<EmbeddedVideo 
+  src="https://www.youtube.com/embed/7BhzaEVqsS4"
+  title="Metrics Tutorial: Measuring User Page Clicks"
+  allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+/>
+
+## Overview
+
+This tutorial guides you through creating a graph to measure and analyze user engagement metrics. By following these steps, you'll be able to track user interactions with your application and gain insights into user behavior and trends.
+
+## Step-by-step Guide
+
+### 1. Select the Data Source
+
+Begin by choosing the source of the metrics. For user engagement, we'll use user sessions as our data source. This will provide us with rich information about how users interact with our application.
+
+### 2. Choose the Graph Type
+
+Next, select the type of graph you want to use to visualize your data. For this example, we'll use a bar graph, which is excellent for comparing values across different categories.
+
+### 3. Apply Filters
+
+To focus on specific user interactions, apply filters to your data. In this case, we'll filter for sessions where the URL contains the word "session". This helps us concentrate on particular pages or features of interest.
+
+### 4. Group the Data
+
+Group the sessions by a relevant attribute. For this example, we'll group by the email that a user reports on that session. This allows us to analyze engagement on a per-user basis.
```

Comment:
Idea: Do we know if most users will have an email attribute associated with their session? I know they must have an `identifier` of the session so not sure we should call out alternatives if a session does not have an email field.